locale/en/blog/vulnerability/jan-2022-security-releases.md

```diff
@@ -1,12 +1,82 @@
 ---
-date: 2022-01-04T16:00:00.000Z
+date: 2022-01-10T16:00:00.000Z
```

Comment:
```suggestion
date: 2022-01-11T00:00:00.000Z
```

Reply:
I think it should be a bit later

date: 2022-01-11T50:00:00.000Z

Reply:
no that's wrong

I mean

date: 2022-01-11T00:50:00.000Z

Reply:
Looks good to me now